### ISSUE TITLE
problem with stringtie prepDE.py

### ISSUE THREAD

**Issue by Pallavimore** [gpertea/stringtie#246]
I am using stringtie (version 2.0) prepDE.py script to generate count matrix of genes and transcripts for EdgeR. I mentioned -e and -G option in stringtie command. I have created one tab-separated file of the sample list as mentioned below to run prepDE.py script.

```
sample1	./ballgown/sample1/sample1.gtf
sample2	./ballgown/sample2/sample2.gtf
sample3	./ballgown/sample3/sample3.gtf
```
I am getting following error for second sample.
```
Traceback (most recent call last):
  File "./prepDE_2.0.py", line 257, in <module>
    geneDict[geneIDs[i]][s[0]]+=v[s[0]]
KeyError: 
```

Could anyone please help me to solve the above error? I already looked into issue https://github.com/gpertea/stringtie/issues/238 but it did not solve my purpose.

**Comment by gpertea** [comment]
It does seem to be the same problem that was fixed in v2.0.4. -- unless you ran your samples with stringtie v2.0.4 (the `-e` runs) and you *still* got that error ?

To recap, the problem was that `stringtie -e` (older than v2.0.4) would sometimes generate additional "novel" transcripts which were not supposed to be there, and those would confuse prepDE. 
The proper way to fix that problem at this point is to re-run the `stringtie -e` round with the new version 2.0.4, and only then run prepDE on the resulting .gtf files like before.

**Comment by Pallavimore** [comment]
Thank you so much for a prompt reply. I used stringtie v2.0.4 and it worked for me.
